build/Tests/FunctionalTests/FunctionalTests.vcxproj

```diff
@@ -295,6 +295,7 @@
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\samples\XAMLCatalog\XAMLCatalog\TestEnabledUITextField.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\samples\XAMLCatalog\XAMLCatalog\UIKitControls\UIButtonViewController.m" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\samples\XAMLCatalog\XAMLCatalog\UIKitControls\UIButtonWithControlsViewController.m" />
+    <ClangCompile Include="..\..\..\tests\functionaltests\Tests\UIKitTests\_UIGestureCoordinator.mm" />
```

Comment:
>. [](start = 27, length = 1)

Add $(MSBuildThisFileDirectory) prefix for consistency? If it's me, I'd also manually move to alongside the other \test\functionaltest files - one contiguous block.